### PR TITLE
Updated release notes

### PR DESCRIPTION
**Pull request by cs-abhishek-shukla**

### CODE REVIEW

**Comment by cs-rohit-khune** [comment]
@cs-abhishek-shukla 

- Release notes look good to me
- I made a few changes in [docs/usage.md] as well, please make sure those changes are also reviewed

**Review by cs-dev-chaudhary — APPROVED** [review]

File: docs/usage.md [hunk 1]
```diff
@@ -19,42 +19,63 @@ To add this widget to the detail view of a module record, perform the following
 
 1. Click to open the detailed view of a record, for example an alert.
 
-2. Click **Edit Template** to display its System View Template (SVT).
+    ![](./media/alert-detailed-view.png)
 
-3. Click **Add Widget** and select the **Fields Of Interest** widget.
+2. Click **Edit Template** (![](./media/icon-edit.svg)) to display its System View Template (SVT).
```
Comment (cs-rohit-khune) [inline docs/usage.md:24]: the icon-edit.svg here dose not display properly in dark theme of git. 

image

Please try add some background to make it visible

File: docs/usage.md [hunk 2]
```diff
@@ -19,42 +19,63 @@ To add this widget to the detail view of a module record, perform the following
...
+
+12. Select fields that you want to exclude from the *Others* column from the **Exclude Following Fields** drop-down.
+
+    ![](./media/foi-edit-view-08.png)
+
+13. Click **Save** to save the changes made to the widget
+
+14. Click **Apply Changes** to add the widget to the detail view of the module's record.
 
 ### Fields of Interest Widget View Panel Views
```
Comment (cs-rohit-khune) [inline docs/usage.md:80]: In `Fields of Interest Widget View Panel Views` section, can we add both screenshots from the same alert, just to maintain consistency?

File: docs/usage.md (hunk 1 above)
Comment (cs-abhishek-shukla) [reply docs/usage.md:24]: Oh!. Thanks. Will do

File: docs/usage.md (hunk 2 above)
Comment (cs-abhishek-shukla) [reply docs/usage.md:80]: I thought I used the same alert. I will check and fix

**Review by cs-rohit-khune — APPROVED** [review]
Approved

**Pull request merged**